src/main.js

```diff
@@ -90,6 +90,9 @@ export default class StepZilla extends Component {
 
   // which step are we in?
   checkNavState(currentStep) {
+    if (this.props.onStepChange) {
+      this.props.onStepChange(currentStep);
```

Comment:
shouldn't this function be called with new step index?
this.getPrevNextBtnState(currentStep)?